ConsoleApplicationGoBoard/ConsoleFunctions.cpp

```diff
@@ -0,0 +1,87 @@
+#include "ConsoleFunctions.h"
+#include <iostream>
+#include <string>
+
+using namespace std;
+
+void ConsoleFuntions::PrintBoard(GoBoard &goBoard)
+{
+	board CurrentBoard = goBoard.ReturnGoBoard();
+	uint8_t SizeOfCurrentBoard = goBoard.ReturnGoBoardSize();
+	for (uint8_t x = 0; x < SizeOfCurrentBoard; x++)
+	{
+		for (uint8_t y = 0; y < SizeOfCurrentBoard; y++)
+		{
+			switch (CurrentBoard[x][y])
+			{
+			case stone::EMPTY:
+				cout << "+ ";
+				break;
+			case stone::BLACK:
+				cout << "O ";
+				break;
+			case stone::WHITE:
+				cout << "0 ";
+				break;
+			}
+		}
+		cout << endl;
+	}
+}
+
+void ConsoleFuntions::playerMove(GoBoard &goBoard)
+{
+	while (!GetCoordiatesAndPlay(goBoard))
+	{
+		std::cout << "you cant go there!" << std::endl;
```

Comment:
it stops the player from being able to play in a place that is allready taken :) probably a bad way of doing it